apps/functions/deadline-submissions/test-deadline-submission/function.json

```diff
@@ -11,6 +11,13 @@
 			"type": "http",
 			"name": "res",
 			"direction": "out"
+		},
+		{
```

Comment:
Good idea! Will you need to extend Terraform to make this work?

Reply:
The relevant role is already set up in Terraform so it should just need the binding